cc-visionary/Inventory-App
# [US 4 & 6] As an admin, edit user details (password) and be able to delete account(s)

What to test:
As an admin
- Edit password constraints.
- Whether or not editing the password of the user is working.
- Error messages.
- Delete a user

What to expect:
- Login as username "admin" and password "password" to login as an admin role.
- Proper error messages will be shown.
- Editing the user password should be working as expected.
